packages/shopify-codemod/transforms/ternary-statement-to-if-statement.js

```diff
@@ -0,0 +1,33 @@
+export default function ternaryStatementToIfStatement(file, api, options) {
+  const j = api.jscodeshift;
+  const {expression, statement, statements} = j.template;
+  const printOptions = options.printOptions || {quote: 'single'};
+
+  return j(file.source)
+    .find(j.ConditionalExpression)
+    .filter(p => p.parent.node.type === 'ExpressionStatement' || p.parent.node.type === 'ReturnStatement')
+    .map(p => p.parent)
+    .replaceWith((parent) => {
+      let p;
+      let statement;
+      switch (parent.node.type) {
+      case 'ExpressionStatement':
+        p = parent.get('expression');
+        statement = j.expressionStatement;
+        break;
+      case 'ReturnStatement':
+        p = parent.get('argument');
+        statement = j.returnStatement;
+        break;
+      }
+
+      const consequent = j.blockStatement([statement(p.node.consequent)]);
+      let alternate = null;
```

Comment:
Yeah, I guess it's not a big deal either way, I just like the consistency with the other `let`s but I get that it's different since it might not be assigned at all.